### PR TITLE
Moving show to backoffice

### PR DESCRIPTION
**Pull request by olivierteboul**
this is a first move. we would next to:
- add back the colors
- add back the links

### CODE REVIEW

**Comment by olivierteboul** [comment]
good point. add a test! thanks

**Comment by rth** [comment]
If I run the backoffice locally I get to a login page. How do I create an account (locally). Ideally, we should add the instructions for setting up the backend to dev_install.md

**Comment by fpquintao** [comment]
> If I run the backoffice locally I get to a login page. How do I create an account (locally)? Ideally, we should add the instructions for setting up the backend to dev_install.md

@rth Hi Roman,
The dev_install recommends the fake db populator, that gives you some users you can login with (user and password can be seen in the fake populator script).

**Comment by rth** [comment]
> The dev_install recommends the fake db populator, that gives you some users you can login with (user and password can be seen in the fake populator script).

OK, great. Will make a documentation PR about it.

**Comment by fpquintao** [comment]
> > The dev_install recommends the fake db populator, that gives you some users you can login with (user and password can be seen in the fake populator script).
> 
> OK, great. Will make a documentation PR about it.

Thanks a ton, sir!

File: icubam/backoffice/server_test.py [hunk 1]
```diff
@@ -33,3 +37,14 @@ def test_login_with_error(self):
...
+  def test_homepage_without(self):
+    handlers = [
+      dashboard.ListBedCountsHandler, icus.ListICUsHandler,
+      users.ListUsersHandler, tokens.ListTokensHandler,
+      ]
+    for handler in handlers:
+      with mock.patch.object(handler, 'get_current_user') as m:
+        m.return_value = self.user
+      response = self.fetch(handler.ROUTE, method='GET')
+    self.assertEqual(response.code, 200)
```
Comment (rth) [inline icubam/backoffice/server_test.py:50]: Sounds good!

**Pull request merged**